### DISCUSSION TITLE
对表执行快照报错

### DISCUSSION THREAD

**Discussion by yangunang** [apache/doris#14065] [A - General / Q&A]
doris 版本1.0.0-rc02，创建快照报错  COMMON_ERROR, msg: failed to choose replica to make snapshot for tablet 2858201. visible version: 3, visible version hash: 7812620662613708187，看报错是replica不一致，查看官方文档没找到解决方案，大家有遇到这个问题吗

**Comment by LemonLiTree** [comment]
show tablet 2858201; 看下呢

**Reply by yangunang** [reply]
> show tablet 2858201; 看下呢

看信息是正常的

**Comment by cszgzjy** [comment]
我也遇上这个问题了，请问楼主解决了吗？

**Reply by yangunang** [reply]
没解决，按照官网修复命令，没有效果，使用了另外一个办法，新创建一个原表的bak表，把原表数据同步一份到bak表，再对bak表执行快照，后面可以将bak表名改为正式表

**Reply by MuXueMaster** [reply]
难道是原表的结构与哪个地方记录的不一致？ 你肯定有升级过 doris把？

**Comment by liangjw** [comment]
Catched this same problem, tablet's status  is normal but can not do a backup. Can any one help?

**Comment by MuXueMaster** [comment]
doris 2.0.3  也遇到这个问题！https://ask.selectdb.com/questions/D1XG/failed-to-make-snapshot-for-tablet-990464-on-backend-10005
